### ISSUE TITLE
Misleading Comment in Speed.py

### ISSUE THREAD

**Issue by Picajoluna** [gboeing/osmnx#691]
Hi,

I was following [this guide of yours](https://github.com/gboeing/osmnx-examples/blob/7cb65dbd64b5923a6013a94b72585f27d7a0acfa/notebooks/02-routing-speed-time.ipynb) on creating drive network isochrones (how far one can get in X minutes of driving).
We initially tried using the `add_edge_speeds()` and `add_edge_travel_time()` functions in [speed.py](https://github.com/gboeing/osmnx/blob/master/osmnx/speed.py). The isochrone produced however was quite odd and suggested that we could travel multiple km's in only a few seconds.

I believe the [comment on line 150](https://github.com/gboeing/osmnx/blob/729f2a19c8f62b0159173a679af6b3ae18813ea9/osmnx/speed.py#L150) is incorrect:
```py
    # convert distance km to meters, and speed km per hour to km per second
    distance_km = edges["length"] / 1000
    speed_km_sec = edges["speed_kph"] / (60 * 60)
```

The edge 'length' attribute is actually initially in meters (at least it is in my graph), whereas you're suggesting that it needs a conversion to meters. In reality, by dividing it by 1000, you're converting those meters to km, which then get divided by `speed_km_sec`.

**Comment by gboeing** [comment]
Thanks. Would you like to propose a PR to correct this?

**Comment by gboeing** [comment]
closed by #693
